west.yml

```diff
@@ -103,7 +103,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 4a060cdb789b8e8eec025f7164d77322ad0cc2a5
```

Comment:
why is this required? commit does not give any hint

Reply:
This is to pull in the following PR, which is related: https://github.com/nrfconnect/sdk-nrfxlib/pull/581